client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/navigator-item-group/index.tsx

```diff
@@ -10,12 +11,14 @@ type Props = {
 };
 
 export const NavigatorItemGroup = ( { children, title }: Props ) => {
-	return (
+	return isEnabled( 'pattern-assembler/color-and-fonts' ) ? (
 		<section>
 			<HStack direction="column" alignment="top" spacing="0">
 				<h3 className="pattern-layout__navigator-item-group">{ title }</h3>
```

Comment:
Also, looks like we have to adjust the padding since we made changes to container's padding and margin